openslides/poll/models.py

```diff
@@ -119,10 +122,19 @@ def print_votescast(self):
         return value
 
     def get_percent_base(self):
-        if self.get_percent_base_choice() == "WITHOUT_INVALID" and self.votesvalid and self.votesvalid > 0:
+        if self.get_percent_base_choice() == "WITHOUT_INVALID" and\
+                self.votesvalid and self.votesvalid > 0:
             base = 100 / float(self.votesvalid)
-        elif self.get_percent_base_choice() == "WITH_INVALID" and self.votescast and self.votescast > 0:
+        elif self.get_percent_base_choice() == "WITH_INVALID" and\
+                self.votescast and self.votescast > 0:
             base = 100 / float(self.votescast)
+        # elif self.get_percent_base_choice() == "WITHOUT_ABSTAIN" and\
+        #     self.votesvalid and self.votesvalid > 0:
+        #     try:
+        #         base = 100 / float(self.votesvalid - abstain)
+        #         #TODO get abstains from server
```

Comment:
Ok, now, the motion/assignment pdf has no percent values (only absolute votes). Not a really nice workaround but ok for (as long as we move the pdfs to client side - then we can use the JS code for percent values).

@normanjaeckel What do you think? Maybe do you have a quick solution to get the abstain value on  server side?